tldw_Server_API/tests/Billing/test_storage_quota_guard.py

```
@@ -0,0 +1,295 @@
+from types import SimpleNamespace
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
+from fastapi import HTTPException
+
+from tldw_Server_API.app.api.v1.API_Deps.storage_quota_guard import (
+    guard_storage_quota,
+)
+
+
+def _fake_user(*, user_id: int = 1, org_ids: list[int] | None = None, active_org_id: int | None = None):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_fake_user`

(ANN202)